target/linux/mediatek/image/filogic.mk

```diff
@@ -2686,6 +2686,8 @@ define Device/zbtlink_zbt-z8103ax
   IMAGES += factory.bin
   IMAGE/factory.bin := append-ubi | check-size $$(IMAGE_SIZE)
   IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  DEVICE_COMPAT_VERSION := 1.1
+  DEVICE_COMPAT_MESSAGE := Partition layout has been changed to fit the bootloader
```

Comment:
same as above - this should be added for the `zbtlink_zbt-z8102ax-v2` target

Reply:
Yep, great thanks! It's midnight in +0800 so I'm a bit tired.